account_invoice_merge/models/account_invoice.py

```diff
@@ -202,7 +202,8 @@ def do_merge(self, keep_references=True, date_invoice=False):
                              ('invoice_id', '=', new_invoice_id)])
                         if invoice_line_ids:
                             so_line.write(
-                                {'invoice_lines': [(6, 0, invoice_line_ids)]})
```

Comment:
You should rename then the variable name to `invoice_lines` to avoid misunderstandings